frontend/appflowy_flutter/lib/mobile/presentation/bottom_sheet/show_transition_bottom_sheet.dart

```diff
@@ -0,0 +1,348 @@
+import 'dart:math' as math;
+
+import 'package:appflowy/plugins/base/drag_handler.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:sheet/route.dart';
+import 'package:sheet/sheet.dart';
+
+import 'show_mobile_bottom_sheet.dart';
+
+Future<T?> showTransitionMobileBottomSheet<T>(
```

Comment:
Would it be more reusable to add a bool value named `withTransition` to `showMobileBottomSheet` instead of adding a new function?

Reply:
That's a really good question. What do you think? My thoughts initially is that if we use the same function signature, some parameters will not be used and we have to rely on comments to communicate which parameters are available in one case and the other.